cmake/libssh2/sys/FindLibSSH2.cmake

```diff
@@ -0,0 +1,35 @@
+# - Try to find the libssh2 library
+# Once done this will define
+#
+# LIBSSH2_FOUND - system has the libssh2 library
+# LIBSSH2_INCLUDE_DIR - the libssh2 include directory
+# LIBSSH2_LIBRARY - the libssh2 library name
+
+if(LIBSSH2_INCLUDE_DIR AND LIBSSH2_LIBRARY)
+    set(LibSSH2_FIND_QUIETLY TRUE)
+endif()
+
+find_path(LIBSSH2_INCLUDE_DIR libssh2.h
+        )
+
+find_library(LIBSSH2_LIBRARY NAMES ssh2
+        )
+
+if(LIBSSH2_INCLUDE_DIR)
+    file(STRINGS "${LIBSSH2_INCLUDE_DIR}/libssh2.h" libssh2_version_str REGEX "^#define[\t ]+LIBSSH2_VERSION_NUM[\t ]+0x[0-9][0-9][0-9][0-9][0-9][0-9].*")
+
+    string(REGEX REPLACE "^.*LIBSSH2_VERSION_NUM[\t ]+0x([0-9][0-9]).*$" "\\1" LIBSSH2_VERSION_MAJOR "${libssh2_version_str}")
```

Comment:
this makes me think this is a derived file ( and not just replicating the variables ) -- then will need something in NOTICE

Reply:
You're right, this was a thing I was going to ask about. This is from curl, and we already mention we bundle curl in the LICENSE file. Should this module be mentioned specifically somewhere?

Reply:
Yes, we follow this http://www.apache.org/licenses/LICENSE-2.0.html#redistribution -- which is not technically required but a derived work should be called out in the notice. I can take care of that on merge, though

Reply:
I have updated the NOTICE file.